server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -76,6 +76,9 @@ export class ScribeLambda implements IPartitionLambda {
 	// Seqeunce number of the last summarised op
 	private lastSummarySequenceNumber: number | undefined;
 
+	// Refs of the service summaries generated since the last client generated summary.
+	private validParentSummaries: string[] | undefined;
```

Comment:
I am curious as to what would happen if there is a scribe restart. Will we lose the validParentSummaries?

Reply:
`validParentSummaries` will be getting stored in the Scribe checkpoints, and also in the latest summarie's protocol tree, so it should recover fine